lib/pure/math.nim

```diff
@@ -67,6 +67,11 @@ when defined(c) or defined(cpp):
 
   proc c_signbit(x: SomeFloat): cint {.importc: "signbit", header: "<math.h>".}
 
+  func c_frexp(x: cfloat, exponent: var cint): cfloat {.
```

Comment:
I'm not sure whether I should export them.

Reply:
well previous declaration was broken, so it's debatable, but how about:
* export it but mark it as `deprecated: use frexp`
* add an alias `c_frexp2` that's not exported and not deprecated and used by nim wrapper `frexp`